### ISSUE TITLE
Broke dependency on ember-template-compiler.js

### ISSUE THREAD

**Issue by inexuscore** [toranb/ember-template-compiler#32]
Compiling templates with `grunt`, `gulp` or any other task runner fails, because the `ember-template-compiler.js` is out-dated.

Every time I set up a new development environment I have to replace this file by the one found in the **ember package** itself.

This issue has been around for a long time, and I'm surprised it hasn't been fixed yet. Any thoughts?

**Comment by rwjblue** [comment]
It really isn't a bug here, this package was required up until Ember 1.10 but not since.  You must use the version of the `ember-template-compiler.js` that is distributed with your particular Ember build.

**Comment by inexuscore** [comment]
@rwjblue Yup its not a bug, but could this repo be updated with the latest template compiler? Thanks.

**Comment by rwjblue** [comment]
What I'm saying is, that there is no latest template compiler. Each Ember build needs to be paired with its specific template compiler counterpart (and those are published/released together).

**Comment by rwjblue** [comment]
This blog post was trying to explain this process: http://emberjs.com/blog/2015/02/05/compiling-templates-in-1-10-0.html

I'm not sure if you've seen it, but I'd definitely like to make sure that it gets the right message across...
